extensions/eda/plugins/event_source/azure_service_bus.py

```diff
@@ -24,6 +6,33 @@
 
 from azure.servicebus import ServiceBusClient
 
+DOCUMENTATION = r"""
+---
+short_description: Receive events from an Azure service bus.
+description:
+  - An ansible-rulebook event source module for receiving events from an Azure service bus.
+options:
+  conn_str:
```

Comment:
This plugin is a, let's say "incomplete" state regarding usage and authentication. I think for now we can include a reference to https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-python-how-to-use-queues?tabs=passwordless in order to get the service bus and the connection string.